tests/Api/Responses/Expected/admin/get_order_response.json

```diff
@@ -57,6 +57,7 @@
             "id": @integer@,
             "quantity": 3,
             "unitPrice": 2000,
+            "originalUnitPrice": 2000,
```

Comment:
After consultation with @GSadee `originalUnitPrice` can be included in that response